nodejs/webrtc-hello-again/server.js

```diff
@@ -0,0 +1,257 @@
+const fs = require('fs')
+const express = require('express')
+const BandwidthWebRTC = require('@bandwidth/webrtc')
+const BandwidthVoice  = require('@bandwidth/voice')
+const uuid = require('uuid');
+const dotenv = require('dotenv').config()
+const jwt_decode = require('jwt-decode')
+const app = express();
+const bodyParser = require('body-parser')
+app.use(bodyParser.json());
+
+
+// config
+const port = 3000
+const localDir = __dirname
+const accountId = process.env.ACCOUNT_ID
+
+// Global variables
+BandwidthWebRTC.Configuration.basicAuthUserName = process.env.USERNAME;
+BandwidthWebRTC.Configuration.basicAuthPassword = process.env.PASSWORD;
+var webRTCController = BandwidthWebRTC.APIController;
+
+BandwidthVoice.Configuration.basicAuthUserName = process.env.USERNAME;
+BandwidthVoice.Configuration.basicAuthPassword = process.env.PASSWORD;
+var voiceController = BandwidthVoice.APIController;
+
+// create a map of PSTN calls that will persist
+let calls = new Map()
+
+// track out session ID
+let sessionId = false;
+let callId = false;
+
+
+/**
+ * Serve the single page app
+ */
+app.get('/', (req, res) => {
+  // return the base page
+  res.sendFile(localDir +'/simple_page.html')
+});
```

Comment:
Typically I would do this using Express' built in static file serving. Assuming your HTML file is called `index.html` and in a directory called `public` this would look like:
```
app.use(express.static('public'))
```
https://expressjs.com/en/starter/static-files.html

Reply:
Thanks, I didn't know about that. This replaced the code for serving the sdk too.